Add Salt for configuration management (#703)
theforeman/foreman-documentation: guides/common/modules/proc_salt-creating-a-host-group-with-salt.adoc

[id="salt_guide_creating_a_host_group_with_salt_{context}"]
= Creating a Host Group With Salt

To bundle provisioning and configuration settings, you can create a host group with Salt enabled for managed hosts.

.Procedure
. In the {ProjectWebUI}, navigate to *Configure > Host Groups*.
. Click *Create Host Group*.

> .. Click the *Host Group* tab and select a *Salt Environment* and a *Salt Master*.

Reviewer: ```suggestion
. Click the *Host Group* tab and select a *Salt Environment* and a *Salt Master*.
```

Author: Applied.

> .. On the *Salt States* tab, assign *Salt States* to your host group.

Reviewer: ```suggestion
.. Click the *Salt States* tab, and assign *Salt States* to your host group.
```
I'm assuming the Host Group tab is open by default? If not, maybe conform it to this style also.

Author: Host Group tab is open by default.

Applied to all three "on the X tab" sentences.

> .. Click the *Salt States* tab and assign *Salt States* to your host group.

Reviewer: ```suggestion
. Click the *Salt States* tab and assign *Salt States* to your host group.
```

Author: Applied.

ifdef::katello,orcharhino[]

> .. Click the *Activation Keys* tab and select an activation key containing the Salt Minion client software.

Reviewer: ```suggestion
. Click the *Activation Keys* tab and select an activation key containing the Salt Minion client software.
```

Author: Applied.

endif::[]
. Click *Submit* to save your host group.

ifdef::katello,orcharhino[]
Managed hosts deployed using this host group automatically install and configure the required Salt Minion client software and register with your Salt Master.
endif::[]

> See {ManagingHostsDocURL}creating-a-host-group[Creating a Host Group] in the _Managing Hosts_ guide for more information.

Reviewer: ```suggestion
For more information, see {ManagingHostsDocURL}creating-a-host-group[Creating a Host Group] in the _Managing Hosts_ guide.
```

Author: Applied.
